app-frontend/src/app/components/scenes/sceneList/sceneList.controller.js

```diff
@@ -83,25 +83,27 @@ export default class SceneListController {
     }
 
     downloadModal(scene) {
-        let images = scene.images.map((image) => {
-            return {
-                filename: image.filename,
-                uri: image.sourceUri,
-                metadata: image.metadataFiles || []
-            };
-        });
+        this.sceneService.getDownloadableImages(scene).then(images => {
```

Comment:
I've noticed that if the api is being slow, the user can click the download button a bunch of times which will send multiple requests. Probably want to make so it only sends 1.

Reply:
:+1: